LanguageFeatures/Views/syntax_A07_t02.dart

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A rule for <viewDeclaration> is added to the grammar, along with
+/// some rules for elements used in view declarations:
+///
+/// <viewDeclaration> ::=
+///   'view' 'class' <typeIdentifier> <typeParameters>?
+///       <viewPrimaryConstructor>?
+///       <interfaces>?
+///   '{'
+///     (<metadata> <viewMemberDeclaration>)*
+///   '}'
+///
+/// <viewPrimaryConstructor> ::=
+///   '(' <type> <identifier> ')'
+///
+/// <viewMemberDeclaration> ::=
+///   <classMemberDefinition>
+/// ...
+/// That is, every view declares exactly one instance variable, and it is final.
+/// A primary constructor (as defined in this document) is just an abbreviated
+/// syntax whose desugaring includes a declaration of exactly one final instance
+/// variable.
+/// ```dart
+/// // Using a primary constructor.
+/// view class V1(R it) {}
+///
+/// // Same thing, using a normal constructor.
+/// view class V2 {
+///   final R it;
+///   V2(this.it);
+/// }
+/// ```
+/// There are no special rules for static members in views. They can be declared
+/// and called or torn off as usual, e.g., View.myStaticMethod(42).
+///
+/// @description Checks that it is a compile-time error if a view declaration
+/// has <viewPrimaryConstructor> and declares a static member with the name
+/// equal to the representation name
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=extension-types
+
+view class View1(int id) {
+  static int get id => 42;
+//^^^^^^^^^^^^^^^^^^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+}
+
+view class View2(int id) {
+  static void set id(int val) {}
```

Comment:
Again not an error because of any name clashes. (But please check the 'Class Member Conflicts' to make sure that it is or is not an error to have a static setter `id=` and an instance member named `id` ... I think it is an error, but I'm saving time to get ready to travel today, so I won't look up everything ;-)

Reply:
Thanks!